WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -314,78 +314,36 @@ extension WordPressAuthenticationManager: WordPressAuthenticatorDelegate {
     /// Presents the Login Epilogue, in the specified NavigationController.
     ///
     func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, source: SignInSource?, onDismiss: @escaping () -> Void) {
+        let mainContext = ContextManager.shared.mainContext
 
         // If adding a self-hosted site, skip the Epilogue
         if let wporg = credentials.wporg,
-           let blog = Blog.lookup(username: wporg.username, xmlrpc: wporg.xmlrpc, in: ContextManager.shared.mainContext) {
-
+           let blog = Blog.lookup(username: wporg.username, xmlrpc: wporg.xmlrpc, in: mainContext) {
             if self.windowManager.isShowingFullscreenSignIn {
                 self.windowManager.dismissFullscreenSignIn(blogToShow: blog)
             } else {
                 navigationController.dismiss(animated: true)
             }
-
-            return
-        }
-
-        if PostSignUpInterstitialViewController.shouldDisplay() {
-            self.presentPostSignUpInterstitial(in: navigationController, onDismiss: onDismiss)
             return
         }
 
-        // Present the epilogue view
-        let storyboard = UIStoryboard(name: "LoginEpilogue", bundle: .main)
-        guard let epilogueViewController = storyboard.instantiateInitialViewController() as? LoginEpilogueViewController else {
-            fatalError()
-        }
-
-        let onBlogSelected: ((Blog) -> Void) = { [weak self, weak navigationController] blog in
-            guard let self, let navigationController else {
-                return
-            }
-
-            // If the user just signed in, refresh the A/B assignments
-            ABTest.start()
-
-            self.recentSiteService.touch(blog: blog)
-            self.presentOnboardingQuestionsPrompt(in: navigationController, blog: blog, onDismiss: onDismiss)
-        }
+        let sites = (try? BlogQuery().blogs(in: mainContext)) ?? []
```

Comment:
To be extra cautious, what do you think using `Account.blogs` instead, like the original code?

https://github.com/wordpress-mobile/WordPress-iOS/blob/84dd814ce7c2f28424a2a99d0845003e8c692712/WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift#L522-L525

Reply:
The main difference is that it checks the account, correct?I updated the code to use `lookupDefaultWordPressComAccount` in https://github.com/wordpress-mobile/WordPress-iOS/pull/23469/commits/30cb2bc4a17caf9935cf6e59eeafe46837cfb226 and removed unused `firstBlog()`.

Related to https://github.com/wordpress-mobile/WordPress-iOS/pull/23469#discussion_r1706946844, this flow and the delegate doesn't seem optimal. There is another delegate method `func createdWordPressComAccount(username: String, authToken: String)` that I think we should remove and roll it into `didFinishLogin`/`didFinishLogout`. The authenticator framework should not determine when the app persists the account, and it's currently a two-step process instead of one. But, again, I suggest to address it separately.